libs/pipingapp/src/lib/utils-garden/getTextColor.ts

```diff
@@ -0,0 +1,7 @@
+export const getTextColor = (status: string) => {
+    if (status === 'Pressure test' || status === 'Insulation' || status === 'Complete')
```

Comment:
Is there a better way to handle this, i dislike case sensitive string comparison.

Reply:
The text color was not in use. Removed.